src/riak_kv_qry_worker.erl

```diff
@@ -131,29 +131,31 @@ new_state() ->
 
 run_sub_qs_fn([]) ->
     ok;
-run_sub_qs_fn([{{qry, ?SQL_SELECT{cover_context = undefined} = Q}, {qid, QId}} | T]) ->
-    Table = Q?SQL_SELECT.'FROM',
+run_sub_qs_fn([{{qry, ?SQL_SELECT{cover_context = undefined} = Q1}, {qid, QId}} | T]) ->
+    Table = Q1?SQL_SELECT.'FROM',
     Bucket = riak_kv_ts_util:table_to_bucket(Table),
     %% fix these up too
     Timeout = {timeout, 10000},
     Me = self(),
     KeyConvFn = make_key_conversion_fun(Table),
-    Opts = [Bucket, none, Q, Timeout, all, undefined, {Q, Bucket}, riak_kv_qry_coverage_plan, KeyConvFn],
+    Q2 = convert_query_to_cluster_version(Q1),
```

Comment:
These subqueries were all compiled in `riak_kv_qry:do_select/2` (https://github.com/basho/riak_kv/blob/riak_ts-develop/src/riak_kv_qry.erl#L267). Are they not, because of this, guaranteed to be the current version?